packages/mobile/locales/es-419/nuxNamePin1.json

```diff
@@ -8,7 +8,7 @@
     "title": "Términos y Condiciones",
     "heading1": "Datos y Privacidad",
     "heading2": "Activos de Celo y Cuenta {{appName}}",
-    "privacy": "Al unirte a esta red, nos das permiso para recopilar información anónima tu uso de la aplicación. Además, si verificas tu número de teléfono, se almacenará un resultado de una función hash en la Red de Celo. Más información en <0>{{tosLink}}</0>",
+    "privacy": "Al unirte a esta red, nos das permiso para recopilar información anónima tu uso de la aplicación. Además, si confirmados tu número de teléfono, se almacenará un resultado de una función hash en la Red de Celo. Más información en <0>{{tosLink}}</0>",
```

Comment:
`confirmados` -> `confirmas`. The rest of the text reads a bit weird too but I guess that's out of scope for this PR